auth/api/groups/http/endpoint.go

```diff
@@ -247,7 +247,7 @@ func ListMembersEndpoint(svc auth.Service) endpoint.Endpoint {
 
 func buildGroupsResponseTree(page auth.GroupPage) groupPageRes {
 	groupsMap := map[string]*auth.Group{}
-	// Parents map keeps its array of children.
+	// Parents' map keeps its array of children.
```

Comment:
typo

Reply:
no it is not, this is possesive of plural `parents`

Reply:
`Parents' ` or `Parent's` ?